client/my-sites/plans/wx-medium/index.tsx

```diff
@@ -4,8 +4,10 @@ import {
 	PLAN_WOOEXPRESS_MEDIUM_MONTHLY,
 } from '@automattic/calypso-products';
 import { Button, Card } from '@automattic/components';
+import { SiteDetails } from '@automattic/data-stores';
```

Comment:
Nit: this should import only the type.
```suggestion
import type { SiteDetails } from '@automattic/data-stores';
```

Reply:
This got addressed in 33178c2.